target/linux/ath79/dts/qca9563_glinet_gl-ar750s.dtsi

```diff
@@ -163,15 +176,35 @@
 	};
 };
 
-&eth0 {
+&pcie {
 	status = "okay";
+};
 
-	mtd-mac-address = <&art 0x0>;
-	phy-handle = <&phy0>;
+&uart {
+	status = "okay";
+};
+
+&usb0 {
+	status = "okay";
+
+	vbus-supply = <&usb_vbus>;
+};
+
+&usb1 {
+	status = "okay";
+};
+
+&usb_phy0 {
+	status = "okay";
+};
+
+&usb_phy1 {
+	status = "okay";
 };
 
 &wmac {
 	status = "okay";
+
 	mtd-cal-data = <&art 0x1000>;
 	mtd-mac-address = <&art 0x1002>;
```

Comment:
Not your issue, but isn't 0x1002 read by default, so this line should be obsolete?

Reply:
I'll leave that to those with deeper understanding of the driver to confirm. Easy enough to remove it if that is the case and that default is deemed by those with knowledge of the driver to be "reliably stable" for the future.